### PR TITLE
[RELEASE START] v3.35.0 > develop

### PR DESCRIPTION
**Pull request by HalfbyteHeroes**
Merge release branch into `develop` after starting release `v3.35.0`

⚠️ Please close PR if changes should not be merged (e.g., support version)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
-## [development]
+## [3.35.0]
```
Comment (wasp898) [inline CHANGELOG.md:6]: There should be a space here 👀

Comment (HalfbyteHeroes) [reply CHANGELOG.md:6]: Where? Above of `## [3.35.0]`?

Comment (wasp898) [reply CHANGELOG.md:6]: Yes, above 🙏

Comment (HalfbyteHeroes) [reply CHANGELOG.md]: Thanks! Added in 1b0b0380 (and e5a99b98 for the release branch)

**Review by wasp898 — APPROVED** [review]

**Pull request merged**